server/routes/helpdesk/validators/problemDetailValidator.test.ts

```diff
@@ -0,0 +1,25 @@
+import validateProblemDetail from './problemDetailValidator'
+
+describe('problemDetailValidator', () => {
+  Array.of(`Something bad happened which meant I couldn't continue with ...`).forEach(name => {
+    it(`should validate given a valid name - ${name}`, () => {
```

Comment:
typo on name - should be problem detail?

Reply:
Fixed